pyteal/ast/subroutine.py

```diff
@@ -611,6 +611,23 @@ def method_signature(self, overriding_name: str = None) -> str:
             overriding_name = self.name()
         return f"{overriding_name}({','.join(args)}){self.type_of()}"
 
+    def method_spec(self) -> dict:
```

Comment:
nit: it would be nice to return the result of `sdk_abi.Method.undictify`, i.e. a `Method` object instead. That's the ultimate target of this function anyway, and it makes the types clearer